### PR TITLE
BW-453

### PR DESCRIPTION
**Pull request by jitendraashutec**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -41,7 +41,17 @@
...
-              ]
+              ],
+              "serviceWorker": true,
+              "optimization": true,
+              "outputHashing": "all",
+              "sourceMap": false,
+              "extractCss": true,
+              "namedChunks": false,
+              "aot": true,
+              "extractLicenses": true,
```
Comment (anihalaney) [inline angular.json]: do we need all these for dev?

File: projects/shared-library/src/lib/environments/environment.dev.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@
 import { IConfig } from './iconfig';
 
 export const environment = {
-  production: false
+  production: true
```
Comment (anihalaney) [inline projects/shared-library/src/lib/environments/environment.dev.ts]: is this just for testing?

File: projects/trivia/src/app/user/components/profile-settings/profile-settings.component.ts [hunk 3]
```diff
@@ -155,6 +157,21 @@ export class ProfileSettingsComponent extends ProfileSettings implements OnDestr
     this.tagsArray.controls = [];
     this.enteredTags.forEach(tag => this.tagsArray.push(new FormControl(tag)));
   }
+
+  onSocialProfileIdPaste(e, i) {
```
Comment (anihalaney) [inline projects/trivia/src/app/user/components/profile-settings/profile-settings.component.ts]: try to have types whenever possible - e? i?

File: projects/trivia/src/app/user/components/profile-settings/profile-settings.component.ts [hunk 4]
```diff
@@ -23,6 +23,8 @@ export class ProfileSettingsComponent extends ProfileSettings implements OnDestr
   cropperSettings: CropperSettings;
   notificationMsg: string;
   errorStatus: boolean;
+  pastedDataUrl: String;
+  arrayString;
```
Comment (anihalaney) [inline projects/trivia/src/app/user/components/profile-settings/profile-settings.component.ts]: missing type?

File: angular.json (hunk 1 above)
Comment (milansar) [reply angular.json]: This is not development environment , it is configuration for staging server. its name is dev
when we do development we run ng serve trivia and this setting does not have impact on it

File: projects/shared-library/src/lib/environments/environment.dev.ts (hunk 2 above)
Comment (milansar) [reply projects/shared-library/src/lib/environments/environment.dev.ts]: This for staging server , development without configuration use environment.ts file 
dev is for staging and production is for production

**Pull request merged**